src/csharp/Pester/CoverageLocationVisitor.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Management.Automation.Language;
+
+namespace Pester
+{
+    public class CoverageLocationVisitor : AstVisitor2
```

Comment:
Add some short info about what this class does if possible. The original predicate is somehow (at least for me) easier to grasp what the intent is.

Reply:
And possibly also why we prefer it over the original predicate.